includes/classes/Command.php

```diff
@@ -1129,9 +1129,10 @@ function ( $prefix ) use ( $args ) {
 				break;
 			}
 
-			// VIP: Pass in $last_processed_object_id independent of nobulk
+			// VIP: Pass in $last_processed_object_id independent of nobulk and track it in an option for long indexing processes that may get killed
 			$last_object_array_key    = array_keys( $query['objects'] )[ count( $query['objects'] ) - 1 ];
 			$last_processed_object_id = $query['objects'][ $last_object_array_key ]->ID;
+			update_option( 'vip_es_index_last_processed_id', $last_processed_object_id );
```

Comment:
I don't think we need it autloaded? 
```suggestion
			update_option( 'vip_es_index_last_processed_id', $last_processed_object_id, 'no' );
```

Reply:
Thinking of this landing in upstream - maybe we should rename `vip_es_` to `ep_`?

Reply:
If we landed it upstream, I'd probably add those hooks directly in this PR then to clean up the option. I thought the autoloading would be good to prevent excessive writes?